NF-Tools-Schemas/animal-model/submitAnimalModel.json

```
@@ -0,0 +1,190 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Submit Animal Model",
+  "type": "object",
+  "properties": {
+    "animalModelName": {
+      "type": "string",
+      "title": "What is the name of the animal model?"
+    },
+    "synonyms": {
+      "type": "string",
+      "title": "Are there any synonyms?"
+    },
+    "description": {
+      "type": "string",
+      "title": "Please provide a description of the animal model (~20 words)"
+    },
+    "backgroundStrain": {
+      "type": "string",
+      "title": "What is the background strain?"
+    },
+    "backgroundSubstrain": {
+      "type": "string",
+      "title": "What is the background substrain?"
```

```suggestion
      "title": "What is the background substrain?"
```

Strains and substrains are a little confusing. Suggest adding an example here and for the strain question too

---

Added an example that pops up when you start to type - but maybe need to add a description below as well.